app/__init__.py

```diff
@@ -30,5 +30,7 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .routes import tasks_bp
```

Comment:
Once you start implementing the goals you can import it here and register the blueprint
```suggestion
    from .routes import tasks_bp, goals_bp
```